tox.ini

```diff
@@ -20,12 +20,12 @@ deps =
 
 [testenv:integration]
 deps =
-    git+https://github.com/juju/python-libjuju.git
-    lightkube
+    juju
```

Comment:
What is the reason for this?

Reply:
Good question, but really I shouldn't do it here anyway.  I'll do this in a separate PR